content/posts/5-rad-dla-juniora.mdx

```diff
@@ -0,0 +1,39 @@
+
+## Code Review to twoje najważniejsze źródełko wiedzy!
+
+Gdybym został zapytany z pomocą, czego nauczyłem się najwięcej, odpowiedź byłaby tylko jedna: Code Review. Ten magiczny proces to idealna okazja do spokojnego przemyślenia i przyswojenia wiedzy na temat twoich lub czyichś błędów/propozycji do kodu. Nie traktuj go jako kolejną przeszkodę, którą musisz pokonać, by móc zamknąć taska. Nie zapomnij również, że twoje pomysły, czy komentarze nie są bezwartościowe tylko dlatego, że dopiero zaczynasz, pisz od razu i najwyżej rozpoczniesz dyskusję, która nauczy czegoś ciebie i twoich rozmówców!
+
+## Pytania kieruj na publiczne kanały.
+
+Staraj się zadawać pytania na kanałach publicznych przeznaczonych dla określonej tematyki np. dla backendowców w projekcie. Jest to o wiele efektywniejsza taktyka, niż pisanie osoba po osobie prywatnie, gdyż każdy od razu widzi twoje pytanie i może akurat jest w temacie, przez co pomoc nadejdzie szybciej, pokaże, ze to, o co pytasz, już jest zaimplementowane. Dzięki takiemu postępowaniu nie tylko ty, ale również inne osoby, które w przyszłości będą miały akurat to samo pytanie, zobaczą od razu na nie odpowiedź.
+
+## Nie bądź tylko klepaczem kodu.
+
+Pewnie nie raz słyszałeś, jak kogoś nazywano „klepaczem kodu”. Kim jest taka osoba? W skrócie to po prostu programista, który niezbyt interesuje się rozwojem projektu. Jest zazwyczaj mało pomysłowy na temat nowych dodatków do aplikacji, jego zmiany w kodzie często nie są jakkolwiek przemyślane. Z code review wyciąga jedynie approve by móc zmerge'ować pull requesta i odpalić kolejnego taska, a jego celem jest po prostu przyjść do pracy wyklepać jak najszybciej zadania i do domu. Takie podejście bardzo szybko odbije się na twoim rozwoju. Staraj się poznawać projekt, w którym jesteś, nie tylko od strony technicznej, ale również biznesowej. Zadawaj pytania i staraj się samemu udzielać w różnych wątkach, gdy widzisz taką możliwość. Nikt ci nic nie zrobi za to, że chciałeś dowiedzieć się czegoś więcej o własnym „dziecku”, które rozwijasz!
+
+<Image src="/images/typing-code.gif" alt="typing code" />
+
+## Podsumowanie
+
+Juniorskich rad znalazłoby się o wiele więcej, ale właśnie ta piątka to te, które mi oraz moim znajomym (**serdecznie dziękuje społeczności Frontlive oraz Type of Web za udzielenie odpowiedzi na moje pytania**) od razu wpadały do głów. Tak więc juniorze, trzymaj się mocno, bo ta przygoda nie będzie łatwa, ale jak będziesz stosował się do wyżej wymienionych punktów, to będzie mniej stresująca.
```

Comment:
Powyższe punkty to rzeczy, które sam chciałbym usłyszeć jeszcze kilka miesięcy temu, gdy stawiałem swoje pierwsze kroki jako Junior Frontend Developer. Droga początkującego programisty nie jest usłana różami, ta przygoda nie będzie łatwa, ale dzięki tym poradom, może być nieco mniej stresująca :)

Artykuł autorstwa [Szymona Hrabii](daj tu link do Linkedina swojego)